e2e/specs/st_header.spec.js

```diff
@@ -21,9 +21,18 @@ describe("st.header", () => {
   });
 
   it("displays a header", () => {
-    cy.get(".element-container .stMarkdown h2").should(
-      "contain",
-      "This header is awesome!"
-    );
+    cy.get(".element-container .stMarkdown h2").should("have.length", 2);
```

Comment:
Nit: can we separate this check in a separate `it`? It's repeated twice for no purpose.